package.json

```diff
@@ -28,8 +29,7 @@
   },
   "homepage": "https://github.com/animatedjs/animated#readme",
   "dependencies": {
-    "invariant": "^2.2.0",
-    "normalize-css-color": "^1.0.1"
```

Comment:
seems like we could continue relying on this instead of copying over the implementation